source/dem/pp_contact_force.cc

```diff
@@ -89,12 +89,11 @@ PPContactForce<dim>::update_contact_information(
   // scalar and vector product
   normal_relative_velocity_value =
     contact_relative_velocity * normal_unit_vector;
-  Tensor<1, dim> normal_relative_velocity =
-    normal_relative_velocity_value * normal_unit_vector;
```

Comment:
Why is the normal relative velocity not a tensor anymore?

Reply:
It was not necessary to keep the vector, because it was being used only in the following line.